client/scripts/views/pages/sputnikdaos.ts

```diff
@@ -0,0 +1,103 @@
+import 'pages/sputnikdaos.scss';
+
+import m from 'mithril';
+import _ from 'lodash';
+import { Tag, Table } from 'construct-ui';
+
+import app from 'state';
+import { navigateToSubpage } from 'app';
+
+import PageLoading from 'views/pages/loading';
+import User from 'views/components/widgets/user';
+import Sublayout from 'views/sublayout';
+import { CommunityOptionsPopover } from './discussions';
+import Near from 'controllers/chain/near/main';
+
+interface IDaoInfo {
+  contractId: string;
+  amount: string;
+  name: string;
+  purpose: string;
+  proposalBond: string;
+  proposalPeriod: string;
+  bountyBond: string;
+  bountyPeriod: string;
+  council: string[];
+}
+
+const SputnikDAOsPage : m.Component<{}, { daosRequested: boolean, daosList: IDaoInfo[] }> = {
+  view: (vnode) => {
+    const activeEntity = app.community ? app.community : app.chain;
```

Comment:
check here if `near` or not. if not, redirect to `/:scope`

Reply:
OK, Got it.